lambdas/url_access_patterns/lambda_for_generateURL/index.js

```diff
@@ -0,0 +1,48 @@
+const AWS = require("aws-sdk");
+const DT = require("date-and-time");
+const crypto = require("crypto");
+const dynamo = new AWS.DynamoDB.DocumentClient();
+const Hashids = require("hashids");
+
+function generateUniqueId() {
+  const salt = crypto.randomBytes(16).toString("base64");
+  const uniqueId = new Hashids(salt, 6);
+  const id = uniqueId.encode(Math.ceil(Math.random() * 1000));
+  return id;
+}
+function response(statusCode, message) {
+  return {
+    statusCode: statusCode,
+    body: JSON.stringify(message),
+  };
+}
+
+function generateTimestamp() {
+  const ts = Date.now();
+  const pattern = DT.compile("URL#YYYY-MM-DD-HH-mm-ss");
+  const timestamp = DT.format(new Date(ts), pattern);
+}
+
+exports.generateURL = async (event) => {
+  const urlID = generateUniqueId();
+  const reqBody = JSON.parse(event.body);
+  const timestamp = generateTimestamp();
+  try {
+    await dynamo
+      .put({
+        TableName: "V-Transfer",
+        Item: {
+          PK: `URL#${timestamp}`,
+          SK: `FILE#${reqBody.f_timestamp}`,
+          GS1_PK: `${urlID}`,
+          hash: `${reqBody.hash}`,
```

Comment:
`PK` should be `FILE#${reqBody.f_timestamp}` and `SK` should be `URL#${timestamp}` according to modelling.
Have you done this intentionally?

Reply:
my bad. will fix it.